wiki/Welcome/hu.md

```diff
@@ -1,130 +1,152 @@
 
-*Teljes magyarázat: [Beatmapping](HU:Beatmapping "wikilink")*
+![Chat konzol barátlista][Chat_Console friendlist image]
 
-Az osu! bárki számára lehetővé teszi beatmapek készítését. Ha úgy gondolod kreatív vagy és szívesen tanulsz, akkor neki is láthatsz. A kezdéshez egyszerűen válasz a főmenüben az "Edit" (Szerkesztés) lehetőséget, majd húzd a szám **.mp3** fájlját a szerkesztőbe. Nyomj F5-öt a frissítéshez és már láthatsz is egy üres mapet a te számod címével. A beatmapping azonban egy *nagyon* komplikált folyamat, valamint szenvedélyes is kell legyél vele kapcsolatban, így ezen az oldalon bővebben nem lesz róla szó. A tanulás legjobb módja az, ha magad is kipróbálod. [Ezen a linken](http://osu.ppy.sh/forum/viewtopic.php?f=56&t=60228) hasznos útmutatókat találhatsz a kezdéshez. A Szerkesztés módot egyébként arra is használhatod, hogy közelebbről szemügyre vegyél egy beatmapet, vagy akár csak azt a részét, ahol mindig elbuksz.**Próbáld kerülni a rankolt beatmapek szerkesztését**. mert vissza fognak állni **nem rankolt** állapotba (persze csak helyileg, a te gépeden), és nem fogsz értük pontokat kapni, amíg nem frissíted őket.
+Ezek az alapok, ne légy szégyenlős segítsége kérni akár játékon belül, akár a fórumon. A wiki-t is olvasgathatod, ha jobban meg szeretnéd érteni a játékot.
 
-Hasznos linkek
-==============
+## Beatmapping (Edit)
+_Teljes magyarázat: [Beatmapping](Beatmapping wikilink)_
 
-Itt van egy pár hasznos link az utazásod megkezdéséhez:-
+Az osu! bárki számára lehetővé teszi a beatmap-ek készítését. Ha úgy gondolod kreatív vagy és szívesen tanulsz, akkor neki is láthatsz. A kezdéshez egyszerűen válasz a főmenüben az "Szerkesztés" (Edit) lehetőséget, majd húzd a zene **.mp3** fájlját a szerkesztőbe. Frissül és már láthatod is az üres mapet a te számod címével. A beatmapping azonban egy _nagyon_ komplikált folyamat amihez szenvedély is kell, így ezen az oldalon bővebben nem lesz róla szó. A legjobb mód az, ha te magad is kipróbálod. [Ezen a linken][Beatmappin/modding: Guide Compendium link] hasznos útmutatókat találhatsz a kezdéshez. A Szerkesztés módot egyébként arra is használhatod, hogy közelebbről szemügyre vegyél egy beatmapet, vagy akár csak azt a részét, ahol mindig elbuksz.  Egyébként **Próbáld kerülni a rank-elt beatmapek szerkesztését**, mert vissza fognak állni **nem rank-elt** állapotba (persze csak a te gépeden), és nem fogsz értük pp-t és pontosságot kapni, amíg nem frissíted őket.
```

Comment:
broken link: change `[Ezen a linken][Beatmappin/modding: Guide Compendium link]` to `[Ezen a linken][Beatmapping/Modding: Guide Compendium link]`